src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -614,6 +643,7 @@ const NetworkModificationNodeEditor = () => {
     };
 
     const onOpenDialog = (id) => {
```

Comment:
This function can be removed and you use the item. action like the node menu

Reply:
name changed to onItemClick